Prox/Prox/AppDelegate.swift

```diff
@@ -199,7 +199,7 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
         if response.notification.request.content.categoryIdentifier == "EVENTS" {
             if let eventKey = response.notification.request.content.userInfo[notificationEventIDKey] as? String,
                 let placeKey = response.notification.request.content.userInfo[notificationEventPlaceIDKey] as? String {
-                Analytics.logEvent(event: AnalyticsEvent.EVENT_NOTIFICATION, params: [AnalyticsEvent.PARAM_ACTION: AnalyticsEvent.CLICKED])
+                Analytics.logEvent(event: AnalyticsEvent.EVENT_CLICKED, params: [AnalyticsEvent.PARAM_ACTION: AnalyticsEvent.BACKGROUND])
```

Comment:
I originally felt like we might not want to log if nothing happens, but that's also a good point.